WordPress/Classes/ViewRelated/System/Notices/NoticeStyle.swift

```diff
@@ -29,11 +29,11 @@ public struct NormalNoticeStyle: NoticeStyle {
     public let actionButtonFont: UIFont? = UIFont.systemFont(ofSize: 14.0)
     public let cancelButtonFont: UIFont? = nil
 
-    public let titleColor: UIColor = WPStyleGuide.darkGrey()
-    public let messageColor: UIColor = WPStyleGuide.darkGrey()
-    public let backgroundColor: UIColor = .clear
+    public let titleColor: UIColor = .white
+    public let messageColor: UIColor = .white
+    public let backgroundColor: UIColor = WPStyleGuide.grey700()
```

Comment:
I totally appreciate that this is a bit awkward because we're using a color from the new palette before the app has the new palette fully implemented. After it's implemented, I think `WPStyleGuide`'s color API will either look a lot different or be replaced entirely.

With that in mind, my preference would be to _not_ change `WordPress-iOS-Shared` and instead make `grey 700` in an extension to `UIColor` just for this file (such as [was done in QuickStartListTitleCell](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Blog/QuickStartListTitleCell.swift#L68)).

I'm just going to leave that as a comment so you can decide to agree or ignore me, as you prefer 😆

Reply:
@nheagy Excellent! I understand. This was awkward and I wasn't sure about modifying `WordPress-iOS-Shared`. Since we're already using `UIColor` extensions anyway, I'll do it that way so there's no need to update the pod. Thank you!